pipelined/bevy_render2/src/lib.rs

```diff
@@ -274,6 +267,47 @@ impl Plugin for RenderPlugin {
     }
 }
 
+pub mod options {
```

Comment:
Can we move this to its own file? With few exceptions we use the "filesystem as module structure" model in Bevy.

Reply:
done!